subprojects/docs/src/docs/userguide/workingWithDependencies.adoc

```diff
@@ -15,13 +15,13 @@
 [[working_with_dependencies]]
 == Working with Dependencies
 
-Gradle provides an extensive API for navigating, inspecting and post-processing post-processing metadata and artifacts of resolved dependencies. With this API at hand, you can implement more complex requirements to your build requiring a "runtime view" of dependencies.
+Gradle provides an extensive API for navigating, inspecting and post-processing metadata and artifacts of resolved dependencies. With this API at hand, you can implement more complex requirements to your build requiring a "runtime view" of dependencies.
```

Comment:
❌ you can implement more complex requirements to your build requiring a "runtime view" of dependencies.

That's a mouthful, can we say this simpler?

Reply:
I think it's somewhat redundant. I am going to remove it. Reading the sentence also make me notice that it doesn't flow that well.